nixos/tests/non-switchable-system.nix

```
@@ -0,0 +1,15 @@
+{ lib, ... }:
+
+{
+  name = "non-switchable-system";
+
+  meta.maintainers = with lib.maintainers; [ nikstur ];
+
+  nodes.machine = {
+    system.switch.enable = false;
+  };
+
+  testScript = ''
+    machine.fail("test -e /run/current-system/bin/switch-to-configuration")
```

This assertion makes me nervous.

```suggestion
    machine.succeed("test ! -e /run/current-system/bin/switch-to-configuration")
```